src/styles/pages/_live.scss

```diff
@@ -191,69 +217,89 @@
 
 .w-event-section__column-head {
   width: 100%;
+
+  @include bp(md) {
+    order: -1;
+  }
 }
 
 .w-event-section__column-left {
   align-items: center;
   display: flex;
   flex-flow: column wrap;
   flex-grow: 1;
+  max-width: 512px;
   padding: 0 32px;
   text-align: center;
   width: 100%;
 
+  @include bp(md) {
+    align-items: flex-start;
+    flex-grow: 0;
+    padding-left: 16px;
+    text-align: inherit;
+    width: 37.5%;
+  }
+
   h3 {
     font-size: 16px;
     margin: 0;
   }
 
   p {
-    max-width: 288px;
     margin: 0;
     margin-top: 24px;
-  }
+    max-width: 512px;
 
-  .w-event-link {
-    font-size: 14px;
-    margin-top: 20px;
+    @include bp(md) {
+      max-width: 288px;
+    }
   }
 
   .w-event-buttons {
     margin: 16px 0;
   }
+
+  .w-feedback-button {
+    margin-top: 49px;
+
+    :first-child {
```

Comment:
tiny nit: adding `svg` here might make it more obvious that this is just moving the image around.